src/test/features/claim/routes/check-and-send.ts

```diff
@@ -58,6 +64,248 @@ describe('Claim issue: check and send page', () => {
+          .expect(res => expect(res).to.be.successful.withText('<a href="/claim/interest-date" class="bold">Change <span class="visuallyhidden">when are you claiming interest from?</span></a>'))
+          .expect(res => expect(res).to.be.successful.withText('<a href="/claim/reason" class="bold">Change <span class="visuallyhidden">why you believe you’re owed the money:</span></a>'))
+          .expect(res => expect(res).to.be.successful.withText('<a href="/claim/timeline" class="bold">Change <span class="visuallyhidden">timeline of what happened</span></a>'))
+          .expect(res => expect(res).to.be.successful.withText('<a href="/claim/evidence" class="bold">Change <span class="visuallyhidden">your evidence (optional)</span></a>'))
+          .expect(res => expect(res).to.be.successful.withText('input id="signedtrue" type="checkbox" name="signed" value="true"'))
+          .expect(res => expect(res).to.be.successful.withText('input type="submit" class="button"'))
+      })
+
+      it('should validate individual to soleTrader flow with lable Business and trading name', async () => {
+        draftStoreServiceMock.resolveFind('claim',
+          { defendant: { ...draftStoreServiceMock.sampleClaimDraftObj.defendant, partyDetails: soleTraderDetails } })
+        feesServiceMock.resolveCalculateIssueFee()
+
+        await request(app)
+          .get(ClaimPaths.checkAndSendPage.uri)
+          .set('Cookie', `${cookieName}=ABC`)
+          .expect(res => expect(res).to.be.successful.withText('<a href="/claim/defendant-sole-trader-details" class="bold">Change <span class="visuallyhidden">full name</span></a>'))
+          .expect(res => expect(res).to.be.successful.withText('Business name'))
+          .expect(res => expect(res).to.be.successful.withText('Trading as SoleTrader Ltd.'))
+      })
+
+      it('should validate individual to Company flow with lable Company Name and Contact persion Name', async () => {
+        draftStoreServiceMock.resolveFind('claim',
+          { defendant: { ...draftStoreServiceMock.sampleClaimDraftObj.defendant, partyDetails: companyDetails } })
+        feesServiceMock.resolveCalculateIssueFee()
+
+        await request(app)
+          .get(ClaimPaths.checkAndSendPage.uri)
+          .set('Cookie', `${cookieName}=ABC`)
+          .expect(res => expect(res).to.be.successful.withText('<a href="/claim/defendant-company-details" class="bold">Change <span class="visuallyhidden">full name</span></a>'))
+          .expect(res => expect(res).to.be.successful.withText('Company Ltd.'))
+          .expect(res => expect(res).to.be.successful.withText('Full name'))
+        // ROC-5326 and ROC-5315 test should be added once the jira closed
+      })
+
+      it('should validate individual to organisation flow with lable organisation Name and Contact persion Name', async () => {
```

Comment:
same throughout the rest of the tests below - please fix spelling mistakes and make the tests labels more meaningful

Reply:
done